AlphaSerpentis/status-react
# Update i18n_resources.cljs

[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

(partial) fixes? #8718 

### Summary

[comment]: # (Summarise the problem and how the pull request solves it)
Validated the JSON file as there was a comma in the last line.

##### Functional

- 1-1 chats
- public chats
- group chats
- wallet / transactions
- dapps / app browsing
- account recovery
- new account
- user profile updates
- networks
- mailservers
- fleet
- bootnodes

##### Non-functional

- battery performance
- CPU performance / speed of the app
- network consumption

### Steps to test


- Step 1, change the device language to one that is known to have caused issues (like French)
- Step 2, open Status and wait for it to load
- Step 3, verify if the language has changed to the appropriate language given.



status: wip
